### ISSUE TITLE
Enable 3rd party distribution on Curseforge

### ISSUE THREAD

**Issue by TheInfamousAlk** [r8420/MoreOverlays-updated#62]
Hi! Love your mod. Would you consider enabling 3rd party distribution on curseforge? Having it disabled makes it a bit more inconvenient to get the mod from 3rd party launchers such as Prism. 

For reference, I was downloading "all the mods 9" and there were only 3 mods out of all of them this update that did not have it enabled, which caused me to have to manually download those from the curseforge website. 

Thank you!

**Comment by Silvertail4764** [comment]
Yes please, I 2nd this, the Modrinth version we can download with 3rd parties, curseforge has some weird additional setting you need to allow, APPARENTLY

And since the Modrinth version isnt kept up to date, its a struggle for modpacks to include it at all for newer versions of MC

**Comment by KernfusionGit** [comment]
Is modrinth not getting updates anymore?
